src/components/SettingsDrawer/SettingsDrawer.css

```diff
@@ -1,31 +1,40 @@
 .SettingsDrawer {
+	background: #313045;
+	box-sizing: border-box;
+	color: #FFFFFF;
+	height: 100%;
+	padding: 96px 32px 32px;
 	position: fixed;
-	top: 0;
 	right: -100%;
-	width: 300px;
-	height: 100%;
-	background: #ccc;
+	top: 0;
+	width: 320px;
+	z-index: 3;
 }
 
 .SettingsDrawer.is-open {
 	animation: bounceInRight forwards 1s;
 	right: 0;
 }
 
+.SettingsDrawer.is-open + .overlay {
+	background: #000000;
+	pointer-events: auto;
+}
+
 .SettingsDrawer.is-closed {
 	animation: bounceOutRight forwards 1s;
 	right: 0;
 }
 
 .SettingsDrawer .close-button {
-	position: absolute;
-	top: 10px;
-	right: 10px;
 	background: url(./close.png) no-repeat center;
```

Comment:
Are we keeping the terribly drawn close button? I think it's meant to be the cog

Reply:
Cog to be added in later today